Library/Formula/mpd.rb

```diff
@@ -61,6 +61,7 @@ class Mpd < Formula
   depends_on "opus" => :optional        # Opus support
   depends_on "libvorbis" => :optional
   depends_on "libnfs" => :optional
+  depends_on "mad" => :optional
```

Comment:
The current behavior is to not depend on mad at all. I am trying to make it optionally available, is this not the right way to do that?

I believe the original intent in disabling the dependency on mad was to simplify the build requirements since ffmpeg handles mp3. I have not been able to diagnose my crashes (the same streams work in mplayer with ffmpeg as the decoder) so this seems like a useful workaround for me and maybe for others.